features/steps/datastore_steps.go

```diff
@@ -18,7 +19,11 @@ func (f *SearchReindexAPIFeature) theTaskResourceShouldHaveTheFollowingFieldsAnd
 		return fmt.Errorf("failed to parse table: %w", err)
 	}
 
-	tasksList, err := f.MongoClient.GetTasks(context.Background(), f.Config.DefaultOffset, 1, f.createdJob.ID)
+	options := mongo.Options{
+		Offset: f.Config.DefaultOffset,
+		Limit:  1,
```

Comment:
(question): This limit should be configurable right rather than a hardcoded number ?

Reply:
used to just pick up one document for test purpose